templates/single-tapestry.php

```diff
@@ -9,6 +9,12 @@
  * @return object null
  */
 
+$browser = $_SERVER['HTTP_USER_AGENT'];
+
+if(is_int(strpos($browser, "MSIE")) || is_int(strpos($browser, "Trident/7.0"))) {
+    echo "<html><div>To use the Tapestry Tool, please use a browser other than Internet Explorer.</div></html>";
```

Comment:
I think inserting this `html` tag makes it invalid HTML. Does it work if you remove the html tag?

```suggestion
    echo "<div>To use the Tapestry Tool, please use a browser other than Internet Explorer.</div>";
```